.appveyor.yml

```diff
@@ -47,6 +48,10 @@ install:
   - set PATH=%PATH%;%GOPATH%\bin
   - go get -u github.com/golang/dep/cmd/dep
   - dep version
+  # Install the Android SDK.
+  - ps: Start-FileDownload 'https://dl.google.com/android/repository/sdk-tools-windows-3859397.zip'
+  - 7z x sdk-tools-windows-3859397.zip -o"%ANDROID_HOME%" > nul
```

Comment:
Just FYI, there would also be no need to quote `%ANDROID_HOME%` here unless it contained spaces, but then again we'd need to quote it in other places, too.

Reply:
Removed.